gulp/theme.js

```diff
@@ -23,6 +24,9 @@ function themeCompileJS() {
   return rollup.rollup({
     input: './theme/js/theme.js',
     plugins: [
+      inject({
+        jQuery: 'jquery'
```

Comment:
Zodat jquery-pjax werkt, dit is geen ES6 module maar een IIFE met een global jQuery